packages/gitlab/dev/mock-gitlab/api-v4-v15.7.0.ts

```diff
@@ -5089,6 +5089,983 @@ export const mockedGitlabReqToRes: Record<string, any> = {
             approvals_before_merge: null,
         },
     ],
+    'projects/10174980/releases?': [
```

Comment:
Can you format the file with Prettier ? it makes the file more readable.

Reply:
done